app/views/pages/codeofconduct.html.erb

```diff
@@ -0,0 +1,11 @@
+<% title t('navigation.codeofconduct') %>
+
+<div class="main_message">
+ Code Of Conduct
```

Comment:
So the CoC we use the [Contributor Covenant](https://www.contributor-covenant.org/). It's actually already translated to various languages (which is rad!). I think we should just link directly to those pages.

Here's our copy of it in the repo: https://github.com/ifmeorg/ifme/blob/master/code_of_conduct.md